### ISSUE TITLE
Selecting features

### ISSUE THREAD

**Issue by dae** [google/cargo-raze#234]
Hi there,

Is my understanding correct that currently the only way to select specific features from a crate is to manually edit the generated remote/<crate>.BUILD.bazel file, removing the unwanted features from the list? I currently am using a Cargo.toml file that specifies different features for different platforms, and am wondering whether such a thing can be accomplished with Bazel now that #212 has landed.

**Comment by UebelAndre** [comment]
Hey, does the `additional_flags` field help you here?
https://github.com/google/cargo-raze/blob/master/examples/remote/complicated_cargo_library/Cargo.toml#L23-L26

I struggle to find a way to specify only specify a feature for a specific platform with just `--cfg` flags though. Maybe this is a lack of functionality on `raze`? Can you elaborate more on what you're trying to do and potentially provide some example? 😄 🙏 

Also, There should be no reason to hand edit generated files. Definitely interested in cases where you feel that's necessary.

**Comment by UebelAndre** [comment]
Maybe https://github.com/rust-lang/cargo/issues/1197 would also be relevant here?

**Comment by dae** [comment]
Sorry for the noise - this looks to be mostly a misunderstanding on my part. The Rust code I use is compiled into a Python module, and the Python distribution rules require that openssl is statically linked on Linux. Thinking it would be better to continue to use the system library on macOS, I had updated my Cargo.toml file to be:

```toml
[target.'cfg(linux)'.dependencies.reqwest]
version = "0.10"
features = ["json", "socks", "stream", "native-tls-vendored"]

[target.'cfg(not(linux))'.dependencies.reqwest]
version = "0.10"
features = ["json", "socks", "stream"]
```

This resulted in the Linux build statically linking openssl, and we were using the nightly compiler at the time for other reasons, so I had thought the problem solved. Re-reading that link above, I realize that I had neglected to enable itarget, and so had probably just been vendoring on all platforms instead. Whoops. 

I now see that for simple feature lists not gated by platform, cargo raze is adjusting the crate_features argument to match what is in Cargo.toml. So the only issue here is that cargo raze doesn't support the nightly target-specific resolution rules that are off by default, and that's probably a niche enough feature that it's not worth worrying about at the moment.
